### DISCUSSION TITLE
h5readatt_octave.oct' failed (error 126)

### DISCUSSION THREAD

**Discussion by PA0MHE** [thliebig/openEMS-Project#53] [Q&A Usage]
I'm a newbie and trying to start working with OpenEMS running under Octave.
But running tutorial simulations I get the error: opening the library 'D:\openEMS\matlab\h5readatt_octave.oct' failed (error 126): The specified module could not be found.
The file h5readatt_octave.oct is present.

My system:
Windows10 i5-6500 CPU, no GPU
openEMS 64bit -- version v0.0.35-108-gc651cce
GNU Octave, version 8.1.0

Thanks for your help.

**Comment by ssmulla** [comment]
I am getting the same problem.

**Comment by PA0MHE** [comment]
By me it was solved when I first ran setup.m

**Reply by ssmulla** [reply]
The problem is fixed on Windows, thank you very much.
But it is not solved in Ubuntu 22.04
